datacenter/dtr/2.2/guides/admin/monitor-and-troubleshoot/troubleshoot-with-logs.md

```diff
@@ -4,7 +4,7 @@ keywords: docker, registry, monitor, troubleshoot
 title: Troubleshoot Docker Trusted Registry
 ---
 
-<!-- TODO: review page for v2.2 -->
+## Troubleshooting overlay networks
```

Comment:
We use the normal form instead of gerund.

Troubleshooting -> Troubleshoot
Accessing -> Access
Recovering -> Recover

The rationale is around consistency with the rest of the docs, and also favoring plain language.